### DISCUSSION TITLE
Welcome to unknown Discussions!

### DISCUSSION THREAD

**Discussion by MarianneAnctil** [MarianneAnctil/unknown#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by matteaue** [comment]
<img width="244" alt="image" src="https://user-images.githubusercontent.com/98493048/197864393-b967eed1-e34a-4624-b9a8-19bc3934df50.png">
<img width="33" alt="image" src="https://user-images.githubusercontent.com/98493048/197864493-44cba36d-0445-4cf8-8f3b-e0a8331b675e.png">

**Comment by matteaue** [comment]
![schématisation](https://user-images.githubusercontent.com/98493048/197864297-2798e252-fe18-43b6-bef4-b8a305c0256a.jpg)

**Reply by matteaue** [reply]
Retirer les évènements et styles des artistes suggérés

**Comment by ghost** [comment]
![Accueil_Final - Table](https://user-images.githubusercontent.com/98491387/199128568-d4b01979-189f-4241-9e09-a09ed1371942.png)
![Accueil_Final - mobile](https://user-images.githubusercontent.com/98491387/199128591-b1abc71f-44b2-406a-8fe7-a959a8cc895f.png)
![schematisation](https://user-images.githubusercontent.com/98491387/199128738-e8d8fb79-4584-49df-aa7c-644105cf1fe5.png)
Voici les maquettes table et mobile ainsi que la schématisation
